src/CISDSC/dscresources/CIS_Microsoft_Windows_10_Enterprise_Release_2004/CIS_Microsoft_Windows_10_Enterprise_Release_2004.psd1

```diff
@@ -4,7 +4,7 @@
 RootModule = 'CIS_Microsoft_Windows_10_Enterprise_Release_2004.schema.psm1'
 
 # Version number of this module.
-ModuleVersion = '1.9.3'
+ModuleVersion = '1.9.2.1'
```

Comment:
What versioning style are you using?
 
Semantic Versioning usually is x.y.z and not x.y.z.a

Reply:
CIS uses three part version numbers for their benchmarks and a given schema directly corresponds to a version of the benchmark. We used the fourth part for our versioning so the schema can still be traced back to a specific version of the source documentation. The parent module uses Semantic versioning.